packages/mdc-form-field/index.js

```diff
@@ -14,28 +14,49 @@
  * limitations under the License.
  */
 
-import {MDCComponent} from '@material/base';
+import MDCComponent from '@material/base/component';
 import MDCFormFieldFoundation from './foundation';
+/* eslint-disable no-unused-vars */
+import {MDCSelectionControl} from '@material/selection-control';
+/* eslint-enable no-unused-vars */
 
 export {MDCFormFieldFoundation};
 
+/**
+ * @extends MDCComponent<!MDCFormFieldFoundation>
+ */
 export class MDCFormField extends MDCComponent {
   static attachTo(root) {
     return new MDCFormField(root);
   }
 
+  /** @param {MDCSelectionControl} input */
```

Comment:
`{?MDCSelectionControl}` or `{!MDCSelectionControl}` (not sure if we want to allow or disallow setting this field to `null`)